testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/RecoveryAuthnCodesAuthenticatorTest.java

```diff
@@ -206,16 +205,15 @@ public void testBruteforceProtectionRecoveryAuthnCodes() {
                 long randomNumber = (long)Math.random()*1000000000000L;
                 enterRecoveryAuthnCodePage.enterRecoveryAuthnCode(String.valueOf(randomNumber));
                 enterRecoveryAuthnCodePage.clickSignInButton();
-                WaitUtils.waitForPageToLoad();
                 enterRecoveryAuthnCodePage.assertCurrent();
                 String feedbackText = enterRecoveryAuthnCodePage.getFeedbackText();
-                Assert.assertEquals(feedbackText, "Invalid recovery authentication code");
+                Assert.assertEquals("Invalid recovery authentication code", feedbackText);
```

Comment:
Also changed a couple of asserts here as `expected` and `actual` were flipped around.